config/initializers/rollbar.rb

```diff
@@ -0,0 +1,57 @@
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+
+  # Here we'll disable in 'test':
+  if Rails.env.test? || Rails.env.development?
+    config.enabled = false
+  end
+
+  # By default, Rollbar will try to call the `current_user` controller method
```

Comment:
This sounds like we'll be putting PII in our logs. I would rather we not do that. Is there some compelling reason we must do that? Can we avoid doing that?

Reply:
Right now it indeed logs the email address associated with the user. I think it's useful to know which user is having a problem but agree we should consider what information we log along with the problem report. Would you feel more comfortable if we just logged the internal user id from the db table and not the email at all? Our staff could still look up the user, but it would need to be intentional. (We could really use guidelines).

Reply:
I would feel better about the user ID, yes. (Although if "internal user ID" = "MIT ID" we have to be very careful not to log any other personal information or I think we fall afoul of MIT's info protection guidelines.)